src/components/TournamentList.module.css

```diff
@@ -0,0 +1,44 @@
+.container {
+  display: flex;
+  flex-direction: column;
+
+  width: 30rem;
+
+  padding: 1rem;
+
+  row-gap: .5rem;
+
+  overflow-y: scroll;
+
+  color: #282828;
+  background-color: #EEEEEE;
+
+  font-family: "Helvetica Neue", Arial, Helvetica, sans-serif;
+}
+
+@media screen and (max-width: 800px) {
```

Comment:
Maybe use the same unit here as the width of the container itself?

Reply:
Done.